docs/advanced_usage.rst

```diff
@@ -204,8 +204,8 @@ Logs Injection
 
 .. automodule:: ddtrace.contrib.logging
 
-HTTP layer
-----------
+HTTP tagging
+------------
```

Comment:
@Isabella-Pham do the sections below this (Query String Tracing, Headers Tracing) look reasonable. Now would be a good time to clean them up or add more to them 🙂 (also let's remember to add to this section in #1860)